newdle/export.py

```diff
@@ -27,10 +28,14 @@ def _generate_answers_for_export(newdle):
 
 def export_answers_to_csv(newdle):
     rows = _generate_answers_for_export(newdle)
-    csv = '\n'.join([','.join(row) for row in rows])
     buffer = BytesIO()
-    buffer.write(csv.encode('utf-8-sig'))
+    output = StringIO()
+    writer = csv.writer(output, dialect='unix')
```

Comment:
A minor thing, but I'd prefer to only quote when necessary while the unix dialect quotes everything regardless.. I think you can use `csv.QUOTE_MINIMAL` for that